nxproxy/configure.in

```diff
@@ -1,186 +0,0 @@
-    CXXFLAGS="$CXXFLAGS -I../nx-X11/exports/include"
-    LIBS="$LIBS -L../nx-X11/exports/lib"
-fi
-
-dnl Check whether --with-ipaq was given.
-
-if test "${with_ipaq}" = yes; then
-  echo -e "enabling ipaq configuration"
-  CXX="arm-linux-c++"
-  CC="arm-linux-gcc"
-  unset ac_cv_prog_armcxx
-  unset ac_cv_prog_armcc
-  unset ac_cv_prog_CXXCPP
-  AC_CHECK_PROG([armcxx],["$CXX"],[yes],[no],[$PATH])
-  AC_CHECK_PROG([armcc],["$CC"],[yes],[no],[$PATH])
-  if test $armcxx = "yes" && test $armcc = "yes" ; then
-    ac_cv_prog_CXX="$CXX"
-    ac_cv_prog_CC="$CC"
-  else
-    AC_MSG_ERROR(installation or configuration problem: I cannot find compiler for arm-linux)
-  fi
-else
-  unset ac_cv_prog_CXX
-  unset ac_cv_prog_CC
-  unset ac_cv_prog_CXXCPP
-fi  
-
-dnl Check for programs.
-
-AC_PROG_CXX
-AC_PROG_CC
-AC_LANG_CPLUSPLUS
-
-dnl Check for BSD compatible install.
-
-AC_PROG_INSTALL
-
-dnl Check for extra header files.
-
-AC_PATH_XTRA
-
-dnl Custom addition.
-
-ac_help="$ac_help
-  --with-symbols          give -g flag to compiler to produce debug symbols
-  --with-info             define INFO at compile time to get basic log output
-  --with-valgrind         clean up allocated buffers to avoid valgrind warnings
-  --with-version          use this version for produced libraries"
-
-dnl Check to see if we're running under Cygwin32.
-
-
-dnl Check to see if we're running under FreeBSD.
-AC_DEFUN(nxconf_FreeBSD,
-[AC_CACHE_CHECK(for FreeBSD environment, nxconf_cv_freebsd,
-[AC_TRY_COMPILE(,[return __FreeBSD__;],
-nxconf_cv_freebsd=yes, nxconf_cv_freebsd=no)
-rm -f conftest*])
-FreeBSD=
-test "$nxconf_cv_freebsd" = yes && FreeBSD=yes])
-nxconf_FreeBSD
-
-
-AC_DEFUN(nxconf_CYGWIN32,
-[AC_CACHE_CHECK(for Cygwin32 environment, nxconf_cv_cygwin32,
-[AC_TRY_COMPILE(,[return __CYGWIN32__;],
-nxconf_cv_cygwin32=yes, nxconf_cv_cygwin32=no)
-rm -f conftest*])
-CYGWIN32=
-test "$nxconf_cv_cygwin32" = yes && CYGWIN32=yes])
-nxconf_CYGWIN32
-
-dnl Check for Darwin environment.
-
-AC_DEFUN(nxconf_DARWIN,
-[AC_CACHE_CHECK(for Darwin environment, nxconf_cv_darwin,
-[AC_TRY_COMPILE(,[return __APPLE__;],
-nxconf_cv_darwin=yes, nxconf_cv_darwin=no)
-rm -f conftest*])
-DARWIN=
-test "$nxconf_cv_darwin" = yes && DARWIN=yes])
-nxconf_DARWIN
-
-dnl Check to see if we're running under Solaris.
-
-AC_DEFUN(nxconf_SUN,
-[AC_CACHE_CHECK(for SunOS environment, nxconf_cv_sun,
-[AC_TRY_COMPILE(,[return __sun;],
-nxconf_cv_sun=yes, nxconf_cv_sun=no)
-rm -f conftest*])
-SUN=
-test "$nxconf_cv_sun" = yes && SUN=yes])
-nxconf_SUN
-
-dnl Check whether --with-version was given.
-
-AC_SUBST(LIBVERSION)
-AC_SUBST(VERSION)
-if test "${with_version}" = yes; then
-  VERSION=${ac_option}
-else
-  VERSION=`cat VERSION`
-fi
-echo -e "compiling version ${VERSION}"
-
-LIBVERSION=`echo ${VERSION} | cut -d '.' -f 1`
-
-CXXFLAGS="$CXXFLAGS -DVERSION=\\\"${VERSION}\\\""
-CPPFLAGS="$CPPFLAGS -DVERSION=\\\"${VERSION}\\\""
-
-dnl Check whether --with-symbols or --without-symbols was
```

Comment:
> We're losing the --with-symbols flag and functionality here. Debian builds by default make use of that.

Good catch. And still... If you look at the current build logs, we have "-g" twice in the options string. One stemming from --with-symbols, the other being added by debhelper. So we can drop this one here.